tests/collection/test_sample.py

```diff
@@ -100,6 +117,21 @@ def test_sample_with_overrides():
     assert collection.second.collect()["c"].to_list() == [3, 4, 6]
 
 
+def test_sample_inline_with_overrides():
+    collection = MyInlinedCollection.sample(
+        overrides=[
+            {"b": 4, "second": [{"c": 3}, {"c": 4}]},
+            {"b": 8, "second": [{"c": 6}]},
+        ]
+    )
+    assert collection.first.collect()["a"].to_list() == [0, 1]
+    assert collection.first.collect()["b"].to_list() == [4, 8]
+
+    assert collection.second is not None
+    assert collection.second.collect()["a"].to_list() == [0, 0, 1]
```

Comment:
```suggestion
    assert collection.second.collect()["b"].to_list() == [4, 4, 8]
```
?

Reply:
Ha, it should _not_ be `[4, 4, 8]` because the second member is not inlined 😄 The `b` in the overrides only refers to the first member.

Reply:
I added the inverse test... it fails 1 out of 2**192 times but I guess that's fine 😂